cfgov/templates/wagtailadmin/js/table-block.js

```diff
@@ -344,12 +396,14 @@
                 ui.$hiddenField.val( JSON.stringify( {
                     data:                      this.handsonTable.instance.getData(),
                     column_widths:             this.getColumnWidths(),
+                    sortable_types:            this.getSortableTypes(),
                     first_row_is_table_header: ui.$hasRowHeaderCheckbox.prop( 'checked' ),
                     first_col_is_header:       ui.$hasColHeaderCheckbox.prop( 'checked' ),
                     is_full_width:             ui.$isFullWidthCheckbox.prop( 'checked' ),
                     is_striped:                ui.$isTableStripedCheckbox.prop( 'checked' ),
                     is_stacked:                ui.$isStackedOnMobileCheckbox.prop( 'checked' ),
-                    fixed_col_widths:          ui.$fixedWidthColsCheckbox.prop( 'checked' )
+                    fixed_col_widths:          ui.$fixedWidthColsCheckbox.prop( 'checked' ),
+                    is_sortable:               ui.$isSortableCheckbox.prop( 'checked' )
                 } ) );
```

Comment:
I don't think so, these `ui.` ones are all the settings checkboxes. If anything I'd move the functions to the bottom since they're looking at the tables that are below the settings checkboxes.